api/application.ts

```diff
@@ -58,6 +58,76 @@ export async function createApp() {
     app.use(helmet.noSniff());
     app.use(helmet.frameguard({ action: 'deny' }));
     app.use(helmet.referrerPolicy({ policy: ['origin'] }));
+    app.use(helmet.hidePoweredBy());
+    app.use(helmet.hsts({ maxAge: 28800000 }));
+    app.use(helmet.xssFilter());
+    app.use(helmet.contentSecurityPolicy({
```

Comment:
This is no longer needed and should not be necessary - you should re-evaluate this file because I don't think any of the above is necessary unless it is relevant for activity tracker (which I don't think it is)